src/internal_events/aws_s3.rs

```diff
@@ -72,7 +75,7 @@ pub mod source {
 
     impl<'a> InternalEvent for SqsMessageProcessingFailed<'a> {
         fn emit_logs(&self) {
-            warn!(message = "Failed to process SQS.", %self.message_id, %self.error);
+            warn!(message = "Failed to process SQS message.", %self.message_id, %self.error);
```

Comment:
That's odd. Shouldn't these (before and after) be something like: `message_id = %self.message_id`

Reply:
Compositionally, yes, I think that was the original intent of these lines, so I'll get this fixed.

As an aside, though, these simply end up as normal fields on the event in the form of `"self": { "message_id": "...", ... }`, so it does technically work, funny enough.

Reply:
Addressed in b1291cb.